h2/src/test/org/h2/test/jdbcx/TestConnectionPool.java

```diff
@@ -99,7 +98,7 @@ public void call() {
         t.execute();
         long time = System.nanoTime();
         Connection conn2 = null;
-        try {
+        try (Connection conn = man.getConnection()) {
```

Comment:
no, this getConnection call needs to be above the t.execute

Reply:
ok! I've reverted that change.